go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/go-vela/vela-github-release
+
+go 1.15
```

Comment:
noticed that we use 1.17 in the github actions workflows - we should match that here.